src/eko/scale_variations/__init__.py

```diff
@@ -3,6 +3,44 @@
 
 from . import expanded, exponentiated
 
+# Mathematica snippet to compute the scale variation terms
```

Comment:
Why did you put this here as a comment?

Reply:
Since it is a mathematica snippet we already have it in `extras` right? If not maybe we can just put it there, do you agree?